src/provider/index.ts

```diff
@@ -158,6 +158,12 @@ export class Main {
      * @param sender The source of the notification.
      */
     private hydrateNotification(payload: NotificationOptions, sender: Identity): StoredNotification {
+        if (!payload.body) {
```

Comment:
This seemed like it should error, and I wanted to make sure that I could test for this kind of thing. If people thing this should be handled differently I'm happy to discuss.

Reply:
Makes sense. Can we make the error messages consistent though?

Reply:
What do you think the errors should be?

Reply:
Either both "must be specified", or both "must have a value" - don't mind which.

Reply:
Oh, just noticed they were different. I must have changed one without changing the other. Will make them both "must have a value"